data/json/items/gun/50.json

```diff
@@ -60,6 +60,34 @@
     "magazines": [ [ "50", [ "belt50" ] ] ],
     "flags": [ "MOUNTED_GUN" ]
   },
+  {
+    "id": "m2browningunmounted",
```

Comment:
This is significantly lighter than the grenade launcher, but I'm still skeptical that the 60lb gun with a 60" length is going to be remotely manageable.